write-fonts/src/tables/variations.rs

```diff
@@ -39,3 +270,77 @@ impl<'a> FromObjRef<Option<read_fonts::tables::variations::Tuple<'a>>> for Vec<F
             .unwrap_or_default()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn point_pack_words() {
+        let thing = PackedPointNumbers {
+            is_all: false,
+            numbers: vec![1002, 2002, 8408, 12228],
+        };
+
+        let runs = thing.iter_runs().collect::<Vec<_>>();
+        assert!(runs[0].are_words);
+        assert_eq!(runs[0].last_point, 0);
+        assert_eq!(runs[0].points, &[1002, 2002, 8408, 12228]);
+
+        let bytes = crate::dump_table(&thing).unwrap();
```

Comment:
This appears to be a different test whose only reason to be in the same function is using the same testdata. Suggest making a const or fn to get testdata and then having two actually different tests.